docs_user/modules/proc_adopting-image-service-with-nfs-backend.adoc

```diff
@@ -109,16 +105,17 @@ spec:
                   annotations:
                     metallb.universe.tf/address-pool: internalapi
                     metallb.universe.tf/allow-shared-ip: internalapi
-                    metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                    metallb.universe.tf/loadBalancerIPs: 172.17.0.80 <3>
                 spec:
                   type: LoadBalancer
           networkAttachments:
           - storage
 EOF
 ----
 +
-* Replace `<ip_address>` with the IP address that you use to reach the `nfs-server`.
-* Replace `<exported_path>` with the exported path in the `nfs-server`.
+<1> Replace `<exported_path>` with the exported path in the `nfs-server`.
```

Comment:
Would it be helpful to include an example of an exported path? Or they vary too much?